packages/loopring_v3/contracts/core/impl/BlockVerifier.sol

```diff
@@ -17,11 +17,13 @@ contract BlockVerifier is ReentrancyGuard, IBlockVerifier
     struct Circuit
     {
         bool registered;
+        uint registeredTime;
```

Comment:
will you deploy a new BlockVerifier or upgrade an existing deployment?

Reply:
blockverifier is the same case

Reply:
But you are modifying the code, so what did you mean "is the same"?